src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp

```diff
@@ -2241,20 +2240,14 @@ std::set<int> EditModeConstraintCoinManager::detectPreselectionConstr(const SoPi
                         // the interval [0 1] Then this result is converted to pixels using the
                         // scale factor.
 
-                        SbVec3f absPos;
-                        SbVec3f trans;
-                        float scaleFactor;
-
                         auto translation = static_cast<SoZoomTranslation*>(
                             static_cast<SoSeparator*>(tailFather)
                                 ->getChild(static_cast<int>(
                                     ConstraintNodePosition::FirstTranslationIndex)));
 
-                        absPos = translation->abPos.getValue();
-
-                        trans = translation->translation.getValue();
-
-                        scaleFactor = translation->getScaleFactor();
+                        SbVec3f constrAbsPos = translation->abPos.getValue();
+                        SbVec3f constrTranslation = translation->translation.getValue();
+                        float constrScaleFactor = translation->getScaleFactor();

```

Comment:
I don't see the point of renaming all the variables. It just makes reviewing the PR more difficult. Besides adding 'constr' to all var names does not feel necessary it just clutters the code